chart/templates/proxy_injector.yaml

```diff
@@ -80,4 +79,3 @@ spec:
     targetPort: proxy-injector
 ---
```

Comment:
realizing this was already an issue, but now that it's default, mind removing the trailing `---` from this file, for consistency with the other templates?